server/api/member.go

```diff
@@ -61,7 +61,8 @@ func (h *memberHandler) ListMembers(w http.ResponseWriter, r *http.Request) {
 
 func getMembers(svr *server.Server) (*pdpb.GetMembersResponse, error) {
 	req := &pdpb.GetMembersRequest{Header: &pdpb.RequestHeader{ClusterId: svr.ClusterID()}}
-	members, err := svr.GetMembers(context.Background(), req)
+	grpcServer := &server.GrpcServer{Server: svr}
+	members, err := grpcServer.GetMembers(context.Background(), req)
```

Comment:
Can we provide a method for `*server.Server` like `s.GetRPCServer()` so that we can write these two lines into one.

```suggestion
	members, err := svr.GetRPCServer().GetMembers(context.Background(), req)
```

Reply:
em, I think one more line here is trivial, compared to introducing the gRPC concept to `server.Server`.

Reply:
Ok. Fair enough.